packages/console/src/pages/Connectors/components/CreateForm/index.module.scss

```diff
@@ -10,6 +10,7 @@
       max-width: 276px;
       width: 276px;
       height: 96px;
+      overflow: hidden;
```

Comment:
What is this for?

Reply:
Prevent overflow when there are too many lines

Reply:
multi-line ellipsis?

```

display: -webkit-box;
text-overflow: ellipsis;
 -webkit-line-clamp: 2;
 -webkit-box-orient: vertical;
 overflow: hidden;
```

Reply:
Yes, and we can even make this a mixin and call it `clamp(n)`

Reply:
FYI `packages/console/src/components/Tooltip/index.module.scss`

<img width="493" alt="image" src="https://user-images.githubusercontent.com/14722250/194849035-7f81c8bf-898a-4a51-8fd6-c5c3ca4eb479.png">

Reply:
updated with a new mixin